plugins/logitech-hidpp/fu-logitech-hidpp-peripheral.c

```diff
@@ -611,6 +611,7 @@ fu_logitech_hidpp_peripheral_setup (FuDevice *device, GError **error)
 	if (idx != 0x00) {
 		self->is_updatable = TRUE;
 		fu_device_add_flag (FU_DEVICE (device), FWUPD_DEVICE_FLAG_IS_BOOTLOADER);
+		fu_device_set_protocol (FU_DEVICE (device), "com.logitech.unifyingsigned");
```

Comment:
in `fu-logitech-hidpp-bootloader` there is a regex to tell which bootloaders have signed images or not, can some similar heuristic exist here?

Reply:
I'm wondering if we need to rip off the bandaid and add fu_device_add_protocol...

Reply:
> rip off the bandaid

Yeooooowch! https://github.com/fwupd/fwupd/pull/2957